src/resolvers/mutation/update-deployment-keda-config/index.js

```
@@ -0,0 +1,23 @@
+/*
+ * Update a Deployment KEDA config
+ * @param {Object} parent The result of the parent resolver.
+ * @param {Object} args The graphql arguments.
+ * @param {Object} ctx The graphql context.
+ * @return {KedaConfig} The updated deployment config.
+ */
+export default async function updateDeploymentKedaConfig(parent, args, ctx) {
+  const { deploymentUuid, state } = args;
+
+  return await ctx.prisma.deployment.update({
```

Can this error handle for invalid `deploymentUuid`?

---

yeah we can query and raise not found before updating.

---

@schnie wdyt?

---

but problem that we have similar code here too https://github.com/astronomer/houston-api/blob/3300c9e743e1b340e80ccde2aa7b11568caf0124/src/resolvers/mutation/update-workspace/index.js#L14
looks like @samblackk tries to be consistent.

---

I think we should do something there otherwise we're going to have potential for a bunch of errors in the logs from what I pasted in the comment above.

---

Yea makes sense. Should we do that everywhere? Should it be a decorator or something reusable?

---

yeah, I can make a ticket for that other one that can error out too.  I bet there are others that we missed as well.

---

so we can just improve this PR, and create separate PR for other places IMO